packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -44,6 +45,11 @@ interface IPendingFlush {
 
 type IPendingState = IPendingMessage | IPendingFlushMode | IPendingFlush;
 
+interface IPendingSnapshot {
+    clientId?: string;
```

Comment:
Any chance you can put some comments for each field?
For example, I'm not sure where clientID is undefined vs. not, and what exactly it tracks (i.e. I'd love to better understand from description if it's undefined only when we never connected, or never sent this op, or something else)